client/signup/jetpack-connect/jetpack-login/index.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Gridicon from 'gridicons';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import MainWrapper from '../main-wrapper';
+import Login from 'blocks/login';
+import TwoFactorAuthentication from 'blocks/login/two-factor-authentication';
+import { localize } from 'i18n-calypso';
+import { isTwoFactorEnabled } from 'state/login/selectors';
+
+const JetpackLogin = ( { twoFactorEnabled, translate } ) => {
+	return (
+		<MainWrapper>
+			<div className="jetpack-connect__login">
+				<div className="jetpack-connect__login-header">
+					<Gridicon icon="user-circle" size={ 64 } />
+					<div>{ translate( 'You are signed out' ) }</div>
+				</div>
+				<div className="jetpack-connect__login-container">
+					{
+						twoFactorEnabled
+						? ( <TwoFactorAuthentication /> )
+						: ( <Login
+							title={ translate( 'Sign in to connect to WordPress.com' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Sign in to WordPress.com' )` *ES Score: 9.42*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/login-component)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).